doc/src/build-instructions.md

```diff
@@ -201,6 +199,43 @@ At this point, you can run `idf.py flash` to upload the 3 binaries up to your ES
 
 However, first, we will build a single binary image file containing all of the above 3 binaries, as well as the AtomVM core libraries.  See [Building a Release Image](#building-a-release-image), below.  But first, it is helpful to understand a bit about how the AtomVM partitioning scheme works, on the ESP32.
 
+### Running tests
```

Comment:
Should we have these QEMU tests as a subsection of running tests, and have a "Running tests on generic_unix" section, that is basically the commands we use out of CI?  If this is an annoying request, I can follow up with a separate PR, because it looks like we don't have a section on testing.

Reply:
Currently we have:

Building for Generic Unix at level 2
Running Tests at level 3

Building for ESP32 at level 2
Running Tests at level 3

Although this apparently isn't very clear, especially with the special note for macOS users at level 4.
Yet I believe Running tests is well under each target, isn't it?

Reply:
Yes!